roadmap/triggers.md

```diff
@@ -0,0 +1,24 @@
+# Tekton Triggers Roadmap
+
+This doc describes the roadmap for [Tekton Triggers](https://github.com/tektoncd/triggers).
+
+In 2019 we created a simple system for creating instances of Tekton resources, triggered
+by json payloads sent to HTTP endpoints (Tekton Triggers).
+
+In 2020 we would like to add missing features and then push for
+[Triggers to be Beta and later GA](#beta-and-ga):
+
+We are targetting improving the experience for both end users and operators:
+
+* For end users:
+  * [Pluggable core interceptors](https://github.com/tektoncd/triggers/issues/271)
+  * [Increased expression support in TriggerBindings](https://github.com/tektoncd/triggers/issues/271)
```

Comment:
This link is incorrect; it has the same link as the previous item. I think that it's supposed to link to issue 367: https://github.com/tektoncd/triggers/issues/367

Reply:
whoops, thanks for being thorough @ncskier ! :D